ci/openstack/install-openshift.sh

```diff
@@ -18,3 +18,41 @@ export INVENTORY="$PWD/playbooks/provisioning/openstack/sample-inventory"
 echo INSTALL OPENSHIFT
 
 ansible-playbook --become --timeout 180 --user openshift --private-key ~/.ssh/id_rsa -i "$INVENTORY" ../openshift-ansible/playbooks/byo/config.yml -e @extra-vars.yaml
+
+
+echo Waiting for the router to come up
+for i in $(seq 15); do
+    if ansible -i "$INVENTORY" masters --user openshift --private-key ~/.ssh/id_rsa -m shell -a 'oc get pod | grep router | grep Running'; then
+        echo Router is running
+        break
+    elif ansible -i "$INVENTORY" masters --user openshift --private-key ~/.ssh/id_rsa -m shell -a 'oc get pod | grep router | grep Failed'; then
```

Comment:
You can avoid use shell ansible module and grep as:

```
oc get pods -l deploymentconfig=router -n default
```

Reply:
Silly comment from my side, otherwise LGTM

Reply:
No, that's great. One question though, doesn't this just print out the status? We need the command to fail if it's not Running or Failed. The last grep fails there which gets propagated to Ansible.

Reply:
Yep, you are right, grep is needed (I think jsonpath supports true or false output as well https://github.com/kubernetes/kubernetes/pull/39063 but I've been playing for 10 minutes and couldn't get it :D)